lti_consumer/templates/html/student.html

```diff
@@ -78,12 +78,11 @@ <h3 class="error_message">
     </h3>
 % endif
 
-% if has_score and comment:
     <h4 class="problem-feedback-label">Feedback on your work from the grader:</h4>
     <div class="problem-feedback">
         ## sanitized with bleach in view
         ${comment}
     </div>
-% endif
```

Comment:
Can you add why this conditional block is removed.

Reply:
I was testing some stuff. Fixed now.